metrics/cli/lib/helper/file_helper.dart

```diff
@@ -0,0 +1,44 @@
+// Use of this source code is governed by the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+import 'dart:io';
+
+/// A class that provides methods for working with the [Directory] and [File]s.
+class FileHelper {
+  /// Replaces variables defined by the [environment.keys]
+  /// with the [environment.values] in the given [file].
+  ///
+  /// Does nothing if the given [file] is `null` or does not exist.
+  /// Does nothing if the given [environment] is `null` or empty.
```

Comment:
I think it would be better to throw an `ArgumentError` if the given `file` or `environment` is null. What do you think?